src/harness/security_testcase.py

```diff
@@ -171,6 +172,29 @@ def doTestCipher(self, cipher, client_cert=None, client_key=None):
     with CiphersOverload(self._sas, [cipher], client_cert, client_key):
       self.assertRegistered([device_a])
 
+  def doSasTestCipher(self, cipher, client_cert, client_key, client_url):
+    """Does a cipher test as described in SSS tests 1 to 5 specification.
+
+    Args:
+      cipher: the cipher openSSL string name to test.
+      client_cert: SAS client certificate file in PEM format to use.
```

Comment:
Please update the description of client_cert and client_key. They are actually path to the file and not the contents of the file in PEM format right?